apps/cookbook/src/app/showcase/accordion-showcase/accordion-showcase.component.html

```diff
@@ -1,54 +1,71 @@
 <div class="example">
-  <h2>Accordion</h2>
   <p>
     An accordion is a vertically stacked list of headers that reveal or hide related sections of
     content.
   </p>
+  <h2>Setup</h2>
   <p>
-    <em>
-      <strong>Note:</strong>
-      Accordion uses Angular animations for animating between its collapsed and expanded state. As
-      no other components depend on the package it is not bundled with Kirby for performance reasons
-      and should instead be provided in the application as detailed in the Angular docs for
-      <a href="https://angular.dev/guide/animations#enabling-the-animations-module">
-        enabling the animations module
-      </a>
-      .
-    </em>
+    Accordions uses Angular animations for animating between its collapsed and expanded state. As no
```

Comment:
Let us get this old typo out of the way 😄 
```suggestion
    Accordions use Angular animations for animating between its collapsed and expanded state. As no
```